### DISCUSSION TITLE
How can I use XamlQRCode in windows-only, wpf, .net 5 application?

### DISCUSSION THREAD

**Discussion by dan-roboticarm** [Shane32/QRCoder#329] [Q&A] [answered]
I have an application where I'd really like to use the  XamlQRCode  functionality. It fits perfectly with the nature of my application which is a wpf-based graphic design tool. Unfortunately, when I tried adding the XamlQRCode sample code from the documentation into my project, XamlQRCode  could not be found. The other QRCoder classes were fine. I assume it's because I'm in a .NET 5 environment, but as long as I'm targeting windows and using wpf, it seems like it should still work.

the sample code from the doc that I inserted:
```
QRCodeGenerator qrGenerator = new QRCodeGenerator();
QRCodeData qrCodeData = qrGenerator.CreateQrCode("The text which should be encoded.", QRCodeGenerator.ECCLevel.Q);
XamlQRCode qrCode = new XamlQRCode(qrCodeData); ///// ERROR OCCURS HERE
DrawingImage qrCodeAsXaml = qrCode.GetGraphic(20);
```

from my .csproj file:
```
...
  <PropertyGroup>
    <TargetFramework>net5.0-windows</TargetFramework>
    <OutputType>WinExe</OutputType>
    <WarningLevel>4</WarningLevel>
    <RunPostBuildEvent>OnOutputUpdated</RunPostBuildEvent>
    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
    <UseWindowsForms>true</UseWindowsForms>
    <UseWPF>true</UseWPF>
    <ImportWindowsDesktopTargets>true</ImportWindowsDesktopTargets>
  </PropertyGroup>
...
```
I'm using QRCoder version 1.4.1 as a nuget package. Is there a way to get this to work? please help!
Thanks,
Dan.

**Comment by dan-roboticarm** [comment, marked as answer]
In case anyone else was wondering, the solution for me was to build it myself and include the correct dll with my project instead of relying on nuget. If there's another way, I'd still like to know, but for now, at least, the issue is resolved.

**Reply by codebude** [reply]
For now it's the only solution until QRCoder 1.4.2 will be released.
